### PR TITLE
Fix/gene_filter_ui

### PR DESCRIPTION
**Pull request by maruk0chan**

### CODE REVIEW

**Comment by maruk0chan** [comment]
This branch has:
1. change the delete button to `trash` icon
2. add the gene filter status

File: components/ScreenerView/ScreenerViewGene.vue [hunk 1]
```diff
@@ -116,6 +116,7 @@
   import datasets from '~/refex-sample/datasets.json';
   import geneFilters from '~/refex-sample/gene_filters.json';
   import ScreenerViewGeneFilter from './ScreenerViewGeneFilter.vue';
+import filters from '~/static/filters.json';
```
Comment (gstein314) [inline components/ScreenerView/ScreenerViewGene.vue]: @maruk0chan 
'filters' is declared but its value is never read
do you need it here? please check it.

**Review by gstein314 — COMMENTED** [review]
@maruk0chan 
I made one comment, please check it out

**Comment by maruk0chan** [comment]
> @maruk0chan 
'filters' is declared but its value is never read
do you need it here? please check it. 

No I don't. Thank you for checking. I have already corrected

**Comment by gstein314** [comment]
Thank you!

**Pull request merged**